storage/innobase/fsp/fsp0fsp.cc

```diff
@@ -3478,9 +5225,8 @@ dberr_t fsp_traverse_extents(
   {
     if (!block)
     {
-      block= fsp_get_latched_xdes_page(
-               page_id_t(space->id, last_descr_page_no),
-               mtr, &err);
+      block= fsp_get_latched_page(page_id_t(space->id, last_descr_page_no),
+                                  mtr, &err);
```

Comment:
In the `for` loop that iterates `cur_extent`, we’re repeatedly evaluating the complex macro `FSP_EXTENT_SIZE`. Can we please assign that run-time constant to a local variable?